ECMA2Yaml/ECMA2Yaml/TopicGenerator.cs

```diff
@@ -175,11 +175,12 @@ public static PageViewModel ToPageViewModel(this Models.Type t, ECMAStore store)
             {
                 pv.References.AddRange(tItem.DerivedClasses.Select(c => store.TypesByUid[c].ToReferenceViewModel()));
             }
-            if (t.ReturnValueType != null
```

Comment:
I know it's a bit confusing right now because we mixed the legacy mref logic with new versioned SDP logic, as much as I'd like to get rid of them, we still can't because UWP is still in mref 😭 

So normally we just leave the legacy part of the code untouched and only make incremental change for new SDP logic. In this case, I recommend that we give return value a new versioned model and keep the old model at the same time, so the code works in both ways. Here's a demo commit that can better describe the idea: